great_expectations/datasource/data_connector/configured_asset_gcs_data_connector.py

```diff
@@ -1,7 +1,171 @@
+import logging
+import os
+from typing import List, Optional
+
+logger = logging.getLogger(__name__)
+
+try:
+    from google.cloud import storage
+    from google.oauth2 import service_account
+except ImportError:
+    storage = None
+    service_account = None
+    logger.debug(
+        "Unable to load GCS connection object; install optional google dependency for support"
+    )
```

Comment:
Nitpick: "Google" instead of "google"

Reply:
I very much appreciate nitpicks 😄 

Great catch, thank you!